Donut/tests/test_routes.py

```diff
@@ -7,4 +7,4 @@ def testHome(client):
   rv = client.get(flask.url_for('home'))
 
   assert rv.status_code == 200
-  assert "Hello world!" in rv.data
+# assert "Hello world!" in rv.data
```

Comment:
Maybe it would be better to check that strings like `"<html"` and `"<body"` exist? Not a huge issue, it the status is 200 then it's probably fine. Also code is more readable if you `import httplib` and then use the constants there, like `httplib.OK`, etc.